projects/batfish/src/org/batfish/main/Batfish.java

```diff
@@ -3578,6 +3575,28 @@ public void pushDeltaEnvironment() {
       return configurationData;
    }
 
+   /**
+    * Returns a sorted list of {@link Path paths} contains all files under the directory
+    * indicated by {@code configsPath}. Directories under {@code configsPath} are recursively
+    * expanded but not included in the returned list.
+    *
+    * <p>Temporary files(files start with {@code .} are omitted from the returned list. </p>
+    * <p>Currently skips all symlinks.</p>
```

Comment:
newline between paragraphs.

Also, this part is wrong. It includes symlinks, does not skip them.